wallet/src/main/scala/org/bitcoins/wallet/Wallet.scala

```diff
@@ -499,6 +494,86 @@ abstract class Wallet
     } yield tx
   }
 
+  override def bumpFee(
+      txId: DoubleSha256DigestBE,
+      newFeeRate: FeeUnit): Future[Transaction] = {
+    for {
+      txDbOpt <- transactionDAO.findByTxId(txId)
+      tx <- txDbOpt match {
+        case Some(db) => Future.successful(db.transaction)
+        case None =>
+          Future.failed(
+            new RuntimeException(s"Unable to find transaction ${txId.hex}"))
+      }
+
+      outPoints = tx.inputs.map(_.previousOutput).toVector
+      spks = tx.outputs.map(_.scriptPubKey).toVector
+
+      utxos <- spendingInfoDAO.findByOutPoints(outPoints)
+      _ = require(utxos.size == tx.inputs.size,
+                  "Can only bump fee for our own transaction")
```

Comment:
nit: Might be nice to have two different messages for this: 

1. If the transaction has _zero_ inputs relevant to us, this error message seems fine
2. If the transaction has one or more inputs that are ours, but not _all_ of them are ours, it might be nice to say this too with a customized error message.